test/e2e/run.sh

```diff
@@ -50,6 +50,7 @@ usage() {
     echo "    reinstall_cri_resmgr: If 1, stop running cri-resmgr, reinstall,"
     echo "             and restart it on the VM before starting test run."
     echo "             The default is 0."
+    echo "    omit_agent: if 1, omit installing/starting/cleaning up cri-resmgr-agent."
```

Comment:
Hmm, I would just use non-empty value instead of  `1`. I.e. you don't need to remember what the required values is, only that `omit_agent` is the name of the variable.

Reply:
Me too. But that would not be consistent with the rest of the similar conventions @askervin put in place. For better consistency, I went with the established conventions.

Reply:
Ah, true. Let's go with this 👍 

LGTM